docs/modules/ROOT/pages/tmail-backend/jmap-extensions/jmapLabels.adoc

```diff
@@ -0,0 +1,160 @@
+= JMAP Labels
+:navtitle: JMAP Labels
+
+This extension allows users to manage their labels.
+
+== Additions to the capability object
+
+Servers supporting the JMAP Labels extension need to advertise it through the session capabilities:
+....
+com:linagora:params:jmap:labels
+....
+
+The associated object is empty.
+
+== Label object
+
+We introduce the `Label` object representing the information of a label, which contains these properties:
+
+- **id**: `Id`. Server-set. Identifier of the label object.
+- **displayName**: `String`. The display name of the label, which can be used to display on the UI.
+- **keyword**: `String`. Server-set. The associated keyword that could be used to set on emails.
+- **color**: `Color`|null. Color of the label.
+
+We introduce the `Color` data type which is a `String` in the form of hex color code. It starts with a hashtag `#` and are followed by six letters and/or numbers.
+The first two letters/numbers refer to red, the next two refer to green, and the last two refer to blue.
+The color values are defined in values between 00 and FF (0 to 255 in RGB).
+
+== Label/get
+
+Standard */get* method as described in Section 5.1, RFC8620.
+
+== Label/set
+
+Standard */set* method as described in Section 5.3, RFC8620.
```

Comment:
Not completely. Only create and update are implemented, not destroy part for example :)

Reply:
We may want/can do the destroyed part, no?

BTW I think we should not limit the feature at the spec level just because current implementation detail. e.g currently we find it hard to support destruction, but in the future, we can add that later with no limit from the spec?

Reply:
As said with an extra level of integration we can even provide destroy ;-)

Reply:
Alright :)